actionpack/lib/action_controller/metal/exceptions.rb

```diff
@@ -51,4 +51,7 @@ class UnknownHttpMethod < ActionControllerError #:nodoc:
 
   class UnknownFormat < ActionControllerError #:nodoc:
   end
+
+  class MissingExactTemplate < ActionControllerError #:nodoc:
```

Comment:
Inherit from UnknownFormat, so the new exception is still rescued by anything that currently matches it